src/tests/src/solver/optim-model-filler/test-component-filler-to-master-pb.cpp

```diff
@@ -201,3 +210,34 @@ BOOST_FIXTURE_TEST_CASE(mixed_variable_listed_in_benders_decomposition,
 }
 
 BOOST_AUTO_TEST_SUITE_END()
+
+BOOST_AUTO_TEST_SUITE(add_constraints_to_master_linear_problem)
+
+BOOST_FIXTURE_TEST_CASE(adding_two_constraints_one_sub_one_master_in_sub,
+                        Fixtures::VarTwoSubNoObjConstrOneSubOneMaster)
+{
+    ComponentFiller componentFiller(*component,
+                                    optimEntityContainer,
+                                    scenario_group_repo,
+                                    Location::SUBPROBLEMS,
+                                    &bendersDecomposition);
+
+    componentFiller.addVariables(time_scenario_ctx);
+    componentFiller.addConstraints(time_scenario_ctx);
+    BOOST_CHECK(linear_pb.getConstraints().size() == 1);
+}
+
+BOOST_FIXTURE_TEST_CASE(adding_two_constraints_one_sub_one_master_in_master,
+                        Fixtures::VarTwoSubNoObjConstrOneSubOneMaster)
+{
+    ComponentFiller masterFiller(*component,
+                                 optimEntityContainer,
+                                 scenario_group_repo,
+                                 Location::MASTER,
+                                 &bendersDecomposition);
+
+    masterFiller.addVariables(time_scenario_ctx);
+    masterFiller.addConstraints(time_scenario_ctx);
+    BOOST_CHECK(linear_pb.getConstraints().size() == 1);
```

Comment:
More verbose on failure
```suggestion
    BOOST_CHECK_EQUAL(linear_pb.getConstraints().size(), 1);
```